src/decision/include/LidarDecision.h

```diff
@@ -49,36 +49,50 @@ class LidarObstacle {
     LidarObstacle(std::vector<Reading> readings);
 
     /**
+    * Gets the distance to the rightmost point of the Obstacle
+    *
+    * @ return the distance to the rightmost point of the Obstacle
+    */
+    distance_t getFirstDistance();
+
+    /**
+    * Gets the distance to the leftmost point of the Obstacle
+    *
+    * @ return the distance to the leftmost point of the obstacle
```

Comment:
Still need to remove the spaces between after `@` symbols. ie, `@return` instead of `@ return` and `@param` instead of `@ param`